twentytwentyone-blocks/assets/css/blocks.css

```diff
@@ -243,4 +243,16 @@ h1.wp-block-site-title a:not(:hover):not(:focus):not(:active) {
 
 .wp-block-social-links.is-style-twentytwentyone-social-icons-color .wp-social-link {
 	background: none;
+}
+
+/*--------------------------------------------------------------
+# Table Block
+--------------------------------------------------------------*/
+
+.wp-block-table {
+	min-width: 240px;
+}
+
+.wp-block-table th, .wp-block-table td {
+	border-width: 0;
```

Comment:
Now that I see it here, I wonder if we should be able to set the border-width of the table block via theme.json?

Reply:
Yeah I think that would make sense

Reply:
https://github.com/WordPress/gutenberg/issues/27651